static/templates/new_stream_users.hbs

```diff
@@ -1,38 +1,10 @@
-{{! Client-side Mustache template for rendering users in the stream creation modal.}}
-
-<div id="copy-from-stream-expand-collapse" class="add-user-label">
-    <i class="toggle fa fa-caret-right" aria-hidden="true"></i>
-    <span class="control-label">
-        {{t "Copy from stream" }}
-    </span>
-</div>
-
-<div id="stream-checkboxes">
-    {{#each streams}}
-    <label class="checkbox add-user-label" data-stream-id="{{this.stream_id}}">
-        <input type="checkbox" name="stream" />
-        <span></span>
-        {{this.name}} ( <i class="fa fa-user" aria-hidden="true"></i> {{this.subscriber_count}})
-    </label>
-    {{/each}}
-</div>
-
-<br>
-<input class="add-user-list-filter" name="user_list_filter" type="text"
-  autocomplete="off" placeholder="{{t "Filter" }}" />
-
-
-<div>
-    <a href="#" draggable="false" class="subs_set_all_users">{{t "Check all" }}</a> |
-    <a href="#" draggable="false" class="subs_unset_all_users">{{t "Uncheck all" }}</a>
-</div>
-
-<div id="user-checkboxes">
-    {{#each users}}
-    <label class="checkbox add-user-label" data-user-id="{{this.user_id}}">
-        <input type="checkbox" name="user" {{#if @first}}checked="checked"{{#unless is_admin}} disabled="disabled"{{/unless}}{{/if}}/>
-        <span></span>
-        {{this.full_name}} ({{this.email}})
-    </label>
-    {{/each}}
+<div class="subscriber_list_add float-left">
+    <form class="form-inline">
+        <div class="add_subscribers_container">
+            <div class="pill-container person_picker">
+                <div class="input" contenteditable="true"
+                  data-placeholder="{{t 'Add people who you think who might be interested in this stream.' }}"></div>
```

Comment:
This is probably the wrong language -- we want something like "Select the initial subscribers for this stream".  And we should probably have you appear by default -- since we don't actually let you create an empty stream.